src/main/java/graphql/annotations/GraphQLAnnotations.java

```diff
@@ -390,12 +390,13 @@ protected GraphQLFieldDefinition getField(Field field) throws GraphQLAnnotations
             typeFunction = newInstance(annotation.value());
         }
 
-        GraphQLOutputType type = (GraphQLOutputType) typeFunction.buildType(field.getType(), field.getAnnotatedType());
+        GraphQLOutputType outputType = (GraphQLOutputType) typeFunction.buildType(field.getType(), field.getAnnotatedType());
+        outputType = field.getAnnotation(NotNull.class) == null ? outputType : new GraphQLNonNull(outputType);
```

Comment:
can you use isAnnotationPresent instead of getAnnotation?

Reply:
done